src/ga.js

```diff
@@ -175,11 +182,16 @@ function getCpmDistribution(cpm) {
   return distribution;
 }
 
+exports.prefixedSend = function(){
```

Comment:
This doesn't need to be a function. Just do `exports.prefixedSend = _trackerPrefix + 'send';`